backend/index.js

```diff
@@ -0,0 +1,75 @@
+const express = require("express");
+const app = express();
+const passport = require("passport");
+const LocalStrategy = require('passport-local').Strategy;
+const BearerStrategy = require('passport-http-bearer').Strategy;
+const jwt = require("jwt-simple");
+const cors = require("cors");
+const pool = require("./db");
+const PORT = process.env.PORT || 3001;
+
+app.use(cors());
+app.use(express.json());
+const USER = "admin";
+const PASS = "12345";
+const SECRET = "mysecret";
+
+passport.use(new LocalStrategy((username, password, done) => {
+    if (username === USER && password === PASS){
+      done(null, jwt.encode({username}, SECRET));
+      return;
+    }
+    done(null, false); 
+  }));
+
+app.post('/login', passport.authenticate('local', { session: false }),
+  (req, res) => {
+    res.send({
+     token: req.user,
+    });
+  },
+);
+
+passport.use(new BearerStrategy((token, done) => {
+  try {
+    const { username } = jwt.decode(token, SECRET);
+    if (username === USER) {
+      done(null, username, { scope: 'read' });
+      return;
+    }
+    done(null, false);
+  } catch (error) {
+    done(null, false);
+  }
```

Comment:
this method is responsible for authenticating, so if something is wrong it is here.